snuba/admin/views.py

```diff
@@ -73,6 +75,27 @@ def health() -> Response:
     return Response("OK", 200)
 
 
+@application.route("/migrations/<group>/list")
```

Comment:
Just a thought but would it be easier to return the list of all migrations by group? (basically equivalent to what the `snuba migrations list` command outputs)? There's really hardly any data, and the UI could easily filter them. The groups and ordering of those groups would also be present there so you wouldn't need to fire so many API calls.

Reply:
that would be easier, but the plan is to have some access control so users can only list the allowed groups.